### PR TITLE
chore: add link to the video cls notebook.

### PR DESCRIPTION
**Pull request by sayakpaul**
We recently added a [notebook](https://github.com/huggingface/notebooks/blob/main/examples/video_classification.ipynb) that shows how to fine-tune the VideoMAE model on a custom dataset. This PR adds the notebook link to the model doc. 

Cc: @osanseviero

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks!

File: docs/source/en/model_doc/videomae.mdx [hunk 1]
```diff
@@ -25,6 +25,7 @@ Tips:
 
 - One can use [`VideoMAEFeatureExtractor`] to prepare videos for the model. It will resize + normalize all frames of a video for you.
 - [`VideoMAEForPreTraining`] includes the decoder on top for self-supervised pre-training.
+- You can check out [this notebook](https://github.com/huggingface/notebooks/blob/main/examples/video_classification.ipynb) that shows how to fine-tune a VideoMAE model on a custom dataset.
```
Comment (NielsRogge) [inline docs/source/en/model_doc/videomae.mdx:28]: We now include a separate "Resources" section in the docs of models, see BERT as an example. Would be great to also add that

Comment (sayakpaul) [reply docs/source/en/model_doc/videomae.mdx]: Done. @stevhliu could you take a look?

**Review by NielsRogge — APPROVED** [review]
Thanks for adding!

**Comment by sayakpaul** [comment]
@NielsRogge I don't have the rights to merge the PR. If you have, could you perform the duties?

**Review by stevhliu — APPROVED** [review]
This is super cool, thanks for adding! I think creating a how-to guide out of this notebook would be a great addition to the docs.

**Pull request merged**